apps/prairielearn/public/stylesheets/local.css

```diff
@@ -239,6 +239,14 @@ table td > .badge {
   padding-right: 1.25rem;
 }
 
+/** Ensure that headings in card headers are styled the same as standard card headers. */
+.card-header > :is(h1, h2, h3, h4, h5, h6) {
+  line-height: 1.5rem;
+  margin-bottom: 0;
+  font-size: 1rem;
+  font-weight: 400;
+}
```

Comment:
I deliberately limited this to direct descendants, which means that in a few headers with more complex structure (namely, the issues page), we still have to use the `h6 font-weight-normal mb-0` classes. These don't include the `line-height: 1.5rem` constraint, but visually it looks fine since they already have a bespoke size/layout.